internal/plugin/validators/lua_validator.go

```diff
@@ -108,16 +108,26 @@ func (v *LuaValidator) ValidateSchema(ctx context.Context, pluginSchema string)
 	return pluginName, nil
 }
 
-func validationErr(name string, e error) error {
+func (v *LuaValidator) validationErr(name string, e error) error {
 	if e == nil {
 		return nil
 	}
+	res := validation.Error{}
 	var errMap map[string]interface{}
 	err := json.ProtoJSONUnmarshal([]byte(e.Error()), &errMap)
 	if err != nil {
-		return fmt.Errorf("unmarshal kong plugin validation error: %v", err)
+		v.logger.With(zap.String("plugin", name)).Error("validation error", zap.Error(e))
+		res.Errs = append(res.Errs, &grpcModel.ErrorDetail{
+			Type: grpcModel.ErrorType_ERROR_TYPE_UNSPECIFIED,
```

Comment:
nit: That's the default (zero) value, so you can leave it empty if you want too.

Reply:
Right, thanks!